src/MIDebugEngine.sln

```diff
@@ -56,164 +57,196 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DebugEngineHost", "DebugEng
 		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Debug-PortablePDB|Any CPU.ActiveCfg = Debug-PortablePDB|Any CPU
 		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Debug-PortablePDB|Any CPU.Build.0 = Debug-PortablePDB|Any CPU
+		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Desktop.Debug|Any CPU.ActiveCfg = Desktop.Debug|Any CPU
+		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Desktop.Debug|Any CPU.Build.0 = Desktop.Debug|Any CPU
+		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Desktop.Release|Any CPU.ActiveCfg = Desktop.Release|Any CPU
+		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Desktop.Release|Any CPU.Build.0 = Desktop.Release|Any CPU
 		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Lab.Debug|Any CPU.ActiveCfg = Lab.Debug|Any CPU
 		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Lab.Debug|Any CPU.Build.0 = Lab.Debug|Any CPU
 		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Lab.Release|Any CPU.ActiveCfg = Lab.Release|Any CPU
 		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Lab.Release|Any CPU.Build.0 = Lab.Release|Any CPU
-		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Desktop.Release|Any CPU.ActiveCfg = Desktop.Release|Any CPU
-		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Desktop.Release|Any CPU.Build.0 = Desktop.Release|Any CPU
 		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}.Release|Any CPU.Build.0 = Release|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Debug-PortablePDB|Any CPU.ActiveCfg = Debug|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Debug-PortablePDB|Any CPU.Build.0 = Debug|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Desktop.Debug|Any CPU.ActiveCfg = Desktop.Debug|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Desktop.Debug|Any CPU.Build.0 = Desktop.Debug|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Desktop.Release|Any CPU.ActiveCfg = Desktop.Release|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Desktop.Release|Any CPU.Build.0 = Desktop.Release|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Lab.Debug|Any CPU.ActiveCfg = Lab.Debug|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Lab.Debug|Any CPU.Build.0 = Lab.Debug|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Lab.Release|Any CPU.ActiveCfg = Lab.Release|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Lab.Release|Any CPU.Build.0 = Lab.Release|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{7654CFBB-30DB-4C20-BDE3-A960CBA2036C}.Release|Any CPU.Build.0 = Release|Any CPU
+		{15BCBEF4-1C2B-412B-925B-34A049097E62}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{15BCBEF4-1C2B-412B-925B-34A049097E62}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{15BCBEF4-1C2B-412B-925B-34A049097E62}.Debug-PortablePDB|Any CPU.ActiveCfg = Debug|Any CPU
+		{15BCBEF4-1C2B-412B-925B-34A049097E62}.Debug-PortablePDB|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
I don't think there is any reason to build SSHDebugPS for the PortablePDB config or the Desktop configs